### PR TITLE
Update useItem return type.

### PR DESCRIPTION
**Pull request by liayoo**
Related: https://bitbucket.org/comcomai/nft-server/pull-requests/146

### CODE REVIEW

File: src/store.ts [hunk 1]
```diff
@@ -225,7 +226,7 @@ export default class Store extends AinftBase {
     itemName,
     quantity,
     params,
-  }: ItemUseParams): Promise<void> {
+  }: ItemUseParams): Promise<NftMetadata|undefined> {
```
Comment (platfowner) [inline src/store.ts:229]: can 'null' be used instead?

Comment (liayoo) [reply src/store.ts]: done!

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Review by woomurf — APPROVED** [review]
LGTM, Thanks!

**Comment by liayoo** [comment]
Thanks for the reviews!

**Pull request merged**